hazelcast/src/main/java/com/hazelcast/ringbuffer/impl/client/ReadManyRequest.java

```diff
@@ -64,11 +64,13 @@ public int getClassId() {
     @Override
     protected Object filter(Object response) {
         ReadResultSetImpl readResultSet = (ReadResultSetImpl) response;
+
         int readCount = readResultSet.readCount();
         List<Data> items = new ArrayList<Data>(readCount);
```

Comment:
not related to this PR, but doesn't this create unnecessary large arraylist? shouldn't the line be
```java
List<Data> items = new ArrayList<Data>(readResultSet.size());
```
?